### ISSUE TITLE
[Bug]: Test teardown takes about 10~ seconds

### ISSUE THREAD

**Issue by ValorHeart** [microsoft/playwright#32535]
### Version

1.47.0

### Steps to reproduce

```
test('get started link', async ({ page }) => {
  await page.goto('https://www.honeybook.com/app/login')
});
```

### Expected behavior

Teardown should take a second

### Actual behavior

Teardown takes about 10 seconds

https://github.com/user-attachments/assets/4ba4c2f8-5ccf-4951-8d5c-c7bafa8ee47e

### Additional context

I'm not sure why that specific page `https://www.honeybook.com/app/login` causes that behavior, it is a regular log-in page, and I'll be happy to have a way to debug why it takes so long to 'close' the browser after only navigating to that page.

### Environment

```shell
Fresh playwright init project in TypeScript.

MacBook M3 Pro with latest OS
WebStorm IDE
```

**Comment by dgozman** [comment]
@ValorHeart I cannot reproduce this, the test run consistently finishes in under 5 seconds. Note that I am not using WebStorm, so perhaps this is related to your IDE integration? In any case, I'd recommend to [record a trace](https://playwright.dev/docs/trace-viewer-intro) to see what's going on. Sharing a full repro with us could also help.

**Comment by ValorHeart** [comment]
Thanks @dgozman , I've done the following:
1. Cleared ~/Library/Caches/ms-playwright cache and re-install the browsers
2. Executed the test with --trace on

I've observed that:
* The test ends, and the browser 'closes' after 3~ seconds, but in the Mac dock, the browser stays there further (This happens on multiple Macs around here)
* The tracer doesn't seem to count that process end duration, but the Terminal does; see screenshot:
<img width="1099" alt="image" src="https://github.com/user-attachments/assets/46ba8f16-7b2a-408a-a48a-5cf5b1f1266f">

Attached reproducing project:
[pwtest.zip](https://github.com/user-attachments/files/16963430/pwtest.zip)
1. npm install
2. npx playwright test --trace on

Thanks, Adi

**Comment by dgozman** [comment]
@ValorHeart I agree with your analysis. Unfortunately, there's not much else we can figure out from the logs. Browser is doing something for 20 seconds. I don't think we can reliably help you without a repro, but this also looks like a problem with your specific setup, so unlikely we can repro on our end.

Some random ideas: you are running under rosetta somehow, there is a corporate proxy/firewall/antivirus/etc that slows things down, filesystem is very slow/busy.

**Comment by ValorHeart** [comment]
Thank you @dgozman 🙏
